classes/PodsAPI.php

```diff
@@ -8682,7 +8682,7 @@ public function get_table_info( $object_type, $object, $name = null, $pod = null
 				$transient .= '_pre_init';
 			}
 
-			if ( false !== $_info && is_array( $_info ) ) {
+			if ( false === $_info || !is_array( $_info ) ) {
```

Comment:
Lgtm, but curious, which line did you identify the path where `$_info` becomes truthy, but a non-array?

Reply:
I simply copied logic from L8307, which has the comment "Data was cached, use that"

Reply:
It was introduced here, so I'll assume there is a reason: [e9ad5fc](https://github.com/pods-framework/pods/commit/e9ad5fc7cdcbd63774c7081e38570946a6f74db5)

Reply:
While transient hammering seems to have stopped, on this screen, we still have ~100 of 1-row SQL queries, each with this traceback:

AFTER

```
SELECT item_id, related_item_id, related_field_id     FROM `wp_podsrel`     WHERE           `field_id` = 427525     AND `item_id` IN ( 560046 )         ORDER BY `weight`
```

```
get_post_meta()
wp-includes/post.php:2032
get_metadata()
wp-includes/meta.php:511
apply_filters('get_post_metadata')
wp-includes/plugin.php:208
PodsMeta->get_post_meta()
wp-content/plugins/pods/classes/PodsMeta.php:2556
PodsMeta->get_meta()
wp-content/plugins/pods/classes/PodsMeta.php:3096
Pods->field()
wp-content/plugins/pods/classes/Pods.php:1392
PodsAPI->lookup_related_items()
wp-content/plugins/pods/classes/PodsAPI.php:7769
pods_query()
wp-content/plugins/pods/includes/general.php:47
PodsData::query()
```

But this looks stay exactly the same on both systems and is probably something for another day.

I'd ship this PR in its latest date, tyvm. :taco:

Reply:
> While transient hammering seems to have stopped, on this screen, we still have ~100 of 1-row SQL queries, each with this traceback:

Yeah, this sounds like a job for a separate issue.  It would help if that info includes basic reproduction instructions, including any Pods exports and/or code that can be dropped into a vanilla WordPress/Pods install for local reproduction.